code/LINDA/LINDA_turf_tile.dm

```diff
@@ -54,6 +54,8 @@
 
 	var/atmos_overlay_type = "" //current active overlay
 
+	var/planetary_atmos = FALSE //air will revert to initial_gas_mix over time
```

Comment:
Note to self: remove this. 

Ideally, we do want to port over the planetary_atmos features and some other new turf stuff from /tg/. I'm hesitant to touch too much LINDA code though.

Reply:
clf3 and some other pyrotechnic reagents call a proc that makes giant fireballs that can melt turfs; make sure that it can't turn lavaland into space. I'm not even sure if it should be changing lavaland turfs to their base type either; maybe lavaland turfs should just be immune to the `meltdown` proc.

Reply:
I double checked, and those fireflashes thankfully let turfs handle the fire processing. Turfs will never delete themselves anymore; they'll always change to their respective baseturf. At this point, I think the only thing that will not cause turfs to use their baseturf is if something qdel's it directly, which nothing should do.